### PR TITLE
[DEV APPROVED] 9540 Images missing from static pages

### PR DESCRIPTION
**Pull request by MarcoBgn**
[TP Card](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=board/5682182231901749200&appConfig=eyJhY2lkIjoiNzEyRUZDMzlENEJERUE4QjJBRDVEQzdENUE1MEE2OEIifQ==&boardPopup=bug/9540/silent)

- The `image_tag` was not used and this created issues with the CDN host
- Updated `hero-get-involved.jpg`

### Pry-byebug

- I have replaced `pry-rails` with `pry-byebug`, what do you think? 
  Same behaviour but gives the option to navigate the stack with `next`, `step`, `finish`, `continue` etc. 
  I find it extremely useful, especially to step into Gems and Engines to check the state on the go.

### CODE REVIEW

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There are code changes, but no corresponding tests. Please include tests if this PR introduces any modifications in behavior. If this is a refactor PR, please ignore the message.Refer to our mas standards guides about opening a PR<a href="link-to-the-mas-standards-about-opening-a-pr">here</a></td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by MarcoBgn** [comment]
Good point @tomas-stefano, done :)

File: Gemfile [hunk 1]
```diff
@@ -31,7 +31,7 @@ group :development, :test do
   gem 'dotenv-rails'
   gem 'factory_bot_rails'
   gem 'launchy'
-  gem 'pry-rails'
+  gem 'pry-byebug'
```
Comment (your) [inline Gemfile]: Awesome, thanks for this!

File: app/views/static_pages/be_involved.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="l-constrained">
   <%= render '/components/hero',
-   img_src: '/assets/static_pages/hero-get-involved.jpg',
+   img_src: image_url('static_pages/hero-get-involved.jpg'),
```
Comment (your) [inline app/views/static_pages/be_involved.html.erb]: 👏

**Pull request merged**